sdk/storage/azure-storage-blob-changefeed/README.md

```diff
@@ -0,0 +1,190 @@
+# Azure Storage Blob ChangeFeed client library for Python
+
+Overview
+
+This preview package for Python enable users to get change feed events by page, get all change feed events, get events in a time range, list events with a continuation token
```

Comment:
Just some grammar tweaks:
"This preview package for Python enables users to get blob change feed events. These events can be lazily generated, iterated by page, retrieved for a specific time interval, or iterated from a specific continuation token."

Reply:
this sounds much better!